int32 id
string location
float32 pm10
float32 co
